tests/core/test_import.py

```diff
@@ -1,4 +0,0 @@
-
-
-def test_import():
-    import <MODULE_NAME>  # noqa: F401
```

Comment:
Rather than remove, I'd recommend just fixing this test.  It's a useful smoke test to expose if there's something broken with basic importing.